### PR TITLE
Update conventions documentation for v4.0.0

### PR DESCRIPTION
**Pull request by peterjc**
Addresses the problem I noted in a comment on issue #380

Thanks for submitting a PR!

Please make sure to check for the following items:
- [ ] Add unit tests and integration tests where applicable.  
      If you've added an error code or changed an error code behavior,
      you should probably add or change a test case file under `tests/test_cases/` and add 
      it to the list under `tests/test_definitions.py`.  
      If you've added or changed a command line option,
      you should probably add or change a test in `tests/test_integration.py`.
- [x] Add a line to the release notes (docs/release_notes.rst) under "Current Development Version".  
      Make sure to include the PR number after you open and get one.
   
Please don't get discouraged as it may take a while to get a review.

### CODE REVIEW

**Comment by peterjc** [comment]
Added the command line API documentation update from #386 by @mrshannon

**Comment by samj1912** [comment]
Hey, sorry for the delay in reviews. If you are still interested, can you fix the merge conflicts and update the PR. Thanks!

File: docs/release_notes.rst [hunk 1]
```diff
@@ -66,6 +66,7 @@ Bug Fixes
 
 Bug Fixes
 
+* Update convention support documentation (#386, #393)
```
Comment (samj1912) [inline docs/release_notes.rst]: Looks like you updated the old release notes. Can you add it to the current development version?

Comment (peterjc) [reply docs/release_notes.rst]: Done. In hindsight I am surprised that wasn't part of the merge conflict.

**Pull request merged**